resolver-dns/src/main/java/io/netty/resolver/dns/DnsCache.java

```diff
@@ -49,17 +49,19 @@
      * @param additionals the additional records
      * @return the cached entries
      */
-    List<DnsCacheEntry> get(String hostname, DnsRecord[] additionals);
+    List<? extends DnsCacheEntry> get(String hostname, DnsRecord[] additionals);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1452)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Consider using varargs for methods or constructors which take an array the last parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUseVarargs)